Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -1582,7 +1582,7 @@ void WindowManager::process_key_event(KeyEvent& event)
         return;
     }
 
-    if (event.type() == Event::KeyDown && event.key() == Key_Super) {
+    if (event.type() == Event::KeyDown && event.key() == Key_Super && !m_current_window_stack->active_window()->title().contains("Keyboard Mapper"sv)) {
```

Comment:
I'm not sure if there's currently a better way to do this, but I think it would be nicer if this used some property of the window (that you may need to add), so applications can opt-in to this behaviour.

Reply:
Was thinking the same. Something along the lines of m_should_ignore_super_key and then check that condition